os/net/lwip/src/core/udp.c

```diff
@@ -77,6 +77,7 @@
 #include <net/lwip/snmp.h>
 #include <net/lwip/arch/perf.h>
 #include <net/lwip/dhcp.h>
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
After adding Werror flag we were getting 'ntohs' related error to overcome that error we have added this header file.